package/MDAnalysis/analysis/hbonds/wbridge_analysis.py

```diff
@@ -537,283 +897,375 @@ def _update_water_selection(self):
+                # the result is analysed by analysis_func which will change the output
+                analysis_func(current, output, self.u, **kwargs)
+        else:
+            # make sure no loop can occur
+            if len(current) <= self.order:
+                for node in graph:
+                    # the new hydrogen bond will be added to the existing bonds
+                    new = link_func(current, node)
+                    self._traverse_water_network(graph[node], new, analysis_func, output, link_func, **kwargs)
+
+    def _expand_index(self, index):
+        '''
+        Expand the index into (resname, resid, name).
+        '''
+        atom = self.u.atoms[index]
+        return (atom.resname, atom.resid, atom.name)
 
-            As the _timeseries to timeserie conversion will be deprecated in 1.0
-            this function will automatically lose its value.
-        """
+    def _expand_timeseries(self, entry, output_format=None):
+        '''
+        Expand the compact data format into the old timeseries form.
+        The function takes in the argument `output_format` to see which output format will be chosen.
+        if `output_format` is not specified, the value will be taken from :attr:`output_format`.
+        If `output_format` is 'sele1_sele2', the output will be the old water bridge analysis format::
+
+          # donor from selection 1 to acceptor in selection 2
+          [sele1_index, sele2_index,
+           (sele1_resname, sele1_resid, sele1_name),
+           (sele2_resname, sele2_resid, sele2_name), dist, angle]
+
+        If `output_format` is 'donor_acceptor', the output will be the old hydrogen bond analysis format::
+
+          # From donor to acceptor
+          [donor_index, acceptor_index,
+           (donor_resname, donor_resid, donor_name),
+           (acceptor_resname, acceptor_resid, acceptor_name), dist, angle]
+        '''
+        output_format = output_format or self.output_format
+        # Expand the compact entry into atom1, which is the first index in the output and atom2, which is the second
+        # entry.
+        atom1, heavy_atom1, atom2, heavy_atom2, dist, angle = entry
+        if output_format == 'sele1_sele2':
+            # If the output format is the sele1_sele2, no change will be executed
+            atom1, atom2 = atom1, atom2
+        elif output_format == 'donor_acceptor':
+            # If the output format is donor_acceptor, use heavy atom position to check which is donor and which is
+            # acceptor
+            if heavy_atom1 is None:
+                # atom1 is hydrogen bond acceptor and thus, the position of atom1 and atom2 are swapped.
+                atom1, atom2 = atom2, atom1
+            else:
+                # atom1 is hydrogen bond donor, position not swapped.
+                atom1, atom2 = atom1, atom2
+        else:
+            raise KeyError('Only \'sele1_sele2\' or \'donor_acceptor\' are allowed as output format.')
```

Comment:
Use double quotation marks to get rid of ugly backslash escapes.

Reply:
Double quotations added.